### PR TITLE
uibk calibration files from the last review and some small gazebo changes

### PR DESCRIPTION
**Pull request by edith-langer**

### CODE REVIEW

File: robotino_controller_configuration_gazebo/package.xml [hunk 1]
```diff
@@ -13,6 +13,8 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostopic</build_depend>
   <build_depend>gazebo_ros_control</build_depend>
+  <build_depend>gazebo-plugins</build_depend>
```
Comment (NadiaHG) [inline robotino_controller_configuration_gazebo/package.xml]: why requires `robotino_controller_configuration_gazebo` these depenedencies?

Comment (edith-langer) [reply robotino_controller_configuration_gazebo/package.xml]: good question. I added that some month ago and as far as I can remember, I had to install those two packages manually to make it work. But maybe we don't need them. Should I remove the two packages?

Comment (NadiaHG) [reply robotino_controller_configuration_gazebo/package.xml]: I think this package doesn't require the dependencies.
`robotino_description` depends on gazebo_plugins and `robotino_gazebo_worlds` on gazebo_ros, but `robotino_controller_configuration_gazebo` only contains ros controllers configuration and the nodes related to the controls.

**Comment by edith-langer** [comment]
not sure if this pull request is still needed because the line I changed in ipa-apartment.launch does not exist anymore. It would be just a kind of old calibration for uibk-robot.

**Comment by ipa-nhg** [comment]
is this still needed? the world changes are already merged https://github.com/squirrel-project/squirrel_robotino/pull/201

**Comment by edith-langer** [comment]
I would merge the start_up angle changes. Not sure about the properties file. After transportation from Freiburg back to Innsbruck it won't be that precise anymore, but probably better then the old values (if Innsbruck hasn't changed the robot's setup)

**Pull request merged**